perma_web/perma/celery_tasks.py

```diff
@@ -2265,9 +2265,13 @@ def queue_internet_archive_uploads_for_date(date_string, limit=100):
         )
         try:
             item = InternetArchiveItem.objects.get(identifier=identifier)
-            item.complete = True
-            item.save(update_fields=['complete'])
-            logger.info(f"Found no pending links: marked IA Item {item.identifier} complete.")
+            # Don't mark an item complete if it's yesterday's
+            if timezone.now() - InternetArchiveItem.datetime(f"{date_string} 00:00:00") > timedelta(days=1):
```

Comment:
Cool! That works. I think slightly more idiomatic, since you have the item here, would be to use `item.span.lower`

```
In [16]: item = InternetArchiveItem.objects.filter(span__isempty=False).first()
In [17]: item.identifier
Out[17]: 'daily_perma_cc_2013-09-12'
In [18]: item.span.lower
Out[18]: datetime.datetime(2013, 9, 12, 0, 0, tzinfo=<UTC>
```

Reply:
Thanks!